scripts/O365_Kill_Active_and_Oauth_Sessions/Readme.md

```diff
@@ -0,0 +1,39 @@
+## kill_sessions.ps1
+
+### Use case
+
+Kill active O365 session and Oauth session refresh tokens
+
+### Brief description 
+
+Pointer to Microsoft documentation about killing sessions
+
+### Data sensitivity
+
+None.
+
+### Local environment requirements
+
+Powershell.
```

Comment:
Technically these commands require more than just PowerShell. Possibly add information about the modules required.